### ISSUE TITLE
about data 

### ISSUE THREAD

**Issue by liangjiubujiu** [welch/rasl#1]
和图片放在一起的.mat矩阵是什么，如何得来的呢/
what is the meaning of the matrix (point.mat)? and how to determine the value of it? thx

**Comment by rickllyxu** [comment]
The matrix contains 3 (x,y) coordinates of the according image. These 3 points determine the initial state of the image.

**Comment by liangjiubujiu** [comment]
but these coordinates(x,y) are not the integer numbers that are float numbers ,so I guess that the coordinates may be transformed by a rule .I want to know something about the rule

**Comment by rickllyxu** [comment]
By my experiment two years ago, these coordinates make up a triangle if you connect each other. You will be surprised that this triangle describes the initial rotation of the object in the picture. Note that the object in the paper is **faces** or **windows of a build**. 
I'd like to point out the `.mat` file is just used to initialize the a rotation parameter in ADMM algorithm. I think you would better focus on how to formulate the problem, and understand its optimization technique and algorithms. Any question about the paper is welcome. btw, I am from uestc, too.

**Comment by liangjiubujiu** [comment]
great!  I have a batch of images about light field, how can I get these corresponding .mat.
I want to get .mat ,can you give me some advices？

**Comment by liangjiubujiu** [comment]
just require these coordinates to make up a trangle ? I use opencv and python to find the character points, I am not sure if these character points can replace the points in .mat? or if you have time now, can you give me more details

**Comment by liangjiubujiu** [comment]
rotation parameter in ADMM algorithm is not random right? cause I want to test if the RASL can use to slove the processing of  light field images.

**Comment by rickllyxu** [comment]
Right! just pick three coordinates whose triangle covers the object in picture. For example, the point located on the forehead and two located on the lower jaws are a reasonable coordinates combination.

**Comment by liangjiubujiu** [comment]
I send e-mail to you rickllykqxu@gmail.com,can we talk  in Chinese？HAHA~ my mailbox is  13792590613@163.com

**Comment by liangjiubujiu** [comment]
I find the datas in .mat is not like (1,3),(4,6)... but (44.12,65.32) I wonder why one coordinate are not  an integer but a decimal?

**Comment by liangjiubujiu** [comment]
if objects in my images are not **windows** or **faces**, does RASL work？

**Comment by liangjiubujiu** [comment]
![image](https://user-images.githubusercontent.com/34046667/40427270-c7f555de-5ecf-11e8-8afe-3f0889aa96bf.png)
I found the Matlab code about RASL, In this picture, what does the parameter raslpara.canonicalCoords mean? how to determine it?

**Comment by liangjiubujiu** [comment]
I notice that only two points in .mat，how to determine the other point to make up a triangle?

**Comment by rickllyxu** [comment]
the variable `canonicalImageSize` describes the length and width of the retangle which can cover the whole object. Sorry I forget the meaning of the `canonicalCoords`.
